node_modules/gh-orgunit/tests/test-import.js

```diff
@@ -0,0 +1,806 @@
+                        'shibMapDisplayname': 'displayname cn',
+                        'shibMapEmail': 'mail email eppn'
+                    };
+                    ConfigTestsUtil.assertUpdateConfig(globalAdminClient, app.id, defaultConfiguration, function() {
+
+                        // Log in through shibboleth
+                        var shibbolethId = util.format('shib-%s', _.random(10000));
+                        var displayName = 'Jack McJackerson';
+                        var attributes = {
+                            'email': TestsUtil.generateTestEmailAddress(),
+                            'displayname': displayName
+                        };
+                        AuthTestsUtil.assertShibbolethLogin(app, shibbolethId, attributes, null, function(restClient) {
+                            UsersTestsUtil.assertGetMe(restClient, function(me) {
+                                assert.ok(!me.anon);
+
+                                // Create a new organisational unit
+                                OrgUnitTestsUtil.assertCreateOrgUnit(globalAdminClient, 'Test part', 'part', {'app': app.id}, function(part) {
+
+                                    // Import some data where the organiser will be linked
+                                    var importData = _getTestTree('1part-1module-1series-1event');
+                                    _getNode(importData, 'eid-1111').organisers = [{'shibbolethId': shibbolethId}];
+                                    OrgUnitTestsUtil.assertImportOrgUnit(globalAdminClient, part.id, importData, false, function() {
+
+                                        // Export the data and verify the user is linked on the event
+                                        OrgUnitTestsUtil.assertExportOrgUnit(globalAdminClient, part.id, null, function(exportedData) {
+                                            // Get the event
+                                            var evt = _getNode(exportedData, 'eid-1111');
+
+                                            // Assert the user was linked
+                                            assert.strictEqual(evt.organisers.length, 1);
+                                            assert.strictEqual(evt.organisers[0].id, me.id);
+                                            assert.strictEqual(evt.organisers[0].shibbolethId, shibbolethId);
+                                            assert.strictEqual(evt.organisers[0].displayName, displayName);
+                                            return callback();
+                                        });
+                                    });
+                                });
+                            });
+                        });
+                    });
+                });
+            });
+        });
+    });
+
+    describe('DAO', function() {
+
+        /**
+         * Test that verifies that the import logic uses transactions
+         */
+        it.skip('verify transactions', function(callback) {
```

Comment:
Yes, my bad, I thought I had finished this. The transaction logic requires transaction support in the EventsDAO, SeriesDAO and OrgUnitDAO and I think I wanted to outscope that from this PR.